tests/NuGet.Services.AzureSearch.Tests/SearchService/SearchTextBuilderFacts.cs

```diff
@@ -31,7 +33,39 @@ public void WhenLuceneQuery_TreatsLeadingIdAsPackageId(bool luceneQuery, string
                     LuceneQuery = luceneQuery,
                 });
 
-                Assert.Equal(expected, actual);
+                Assert.Equal(expected, actual.Text);
+            }
+
+            [Theory]
+            [InlineData("id:hello", "hello")]
+            [InlineData("packageid:hello", "hello")]
+            [InlineData("packageid:" + MaxId, MaxId)]
+            public void ExtractsPackageIdWhenThereIsOnlyOneClause(string query, string expected)
```

Comment:
Should we test if the package id is wrapped in quotes?

Reply:
Yeah, good idea.